.github/actions/build.sh

```diff
@@ -72,6 +72,8 @@ do
         else
             echo "$ make arch-${GH_ARCH%%-*}-${GH_ARCH##*-} -C ./spk/${package}" >>build.log
             make arch-${GH_ARCH%%-*}-${GH_ARCH##*-} -C ./spk/${package} |& tee >(tail -15 >>build.log)
+            make arch-${GH_ARCH%%-*}-${GH_ARCH##*-} -C ./spk/${package} clean-source |& tee >(tail -15 >>build.log)
+            make arch-${GH_ARCH%%-*}-${GH_ARCH##*-} -C ./spk/${package} spkclean |& tee >(tail -15 >>build.log)
```

Comment:
why not below, when cleaning is done for all packages except the packages to keep?

```
    if [ "$(echo ${PACKAGES_TO_KEEP} | grep -ow ${package})" = "" ]; then
        # free disk space (but not for packages to keep)
        make -C ./spk/${package} clean
    else
        # delete source folders only for packages to keep
        make -C ./spk/${package} source-clean
    fi
```

Reply:
only seeing your comment now, started a new PR to change this over here as it makes much more sense indeed.  Ref. https://github.com/SynoCommunity/spksrc/pull/6038